pkgs/tools/admin/awsls/default.nix

```diff
@@ -0,0 +1,26 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "awsls";
+  version = "0.11.0";
+
+  src = fetchFromGitHub {
+    owner = "jckuester";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-iy9tohmVUtNXYVfe6pZ+pbbLlcK6Fu1GgzTWMD+3xP0=";
+  };
+
+  vendorSha256 = "sha256-ZyMO+KCqoePF6MqHFt8X4tZR4nBhuSPgJDrX+emM6jc=";
+
+  ldflags = [ "-s" "-w" "-X github.com/jckuester/awsls/internal.version=${version}" "-X github.com/jckuester/awsls/internal.commit=${src.rev}" "-X github.com/jckuester/awsls/internal.date=unknown" ];
```

Comment:
```suggestion
  ldflags = let t = "github.com/jckuester/awsls/internal"; in [ "-s" "-w" "-X ${t}.version=${version}" "-X ${t}.commit=${src.rev}" "-X ${t}.date=unknown" ];
```

Reply:
Hm yes  we could DRY  that up a bit,  although I'm okay with being explicit  here.  Why  did you choose `t` as the variable name?  Maybe  `prefix` would  be  better?

Reply:
IDK, t is short and used in other places

Reply:
Done